lark/utils.py

```diff
@@ -154,47 +172,51 @@ def get_regexp_width(expr):
 _ID_START =    'Lu', 'Ll', 'Lt', 'Lm', 'Lo', 'Mn', 'Mc', 'Pc'
 _ID_CONTINUE = _ID_START + ('Nd', 'Nl',)
 
-def _test_unicode_category(s, categories):
+def _test_unicode_category(s: str, categories: Union[Tuple[str, str, str, str, str, str, str, str], Tuple[str, str, str, str, str, str, str, str, str, str]]) -> bool:
```

Comment:
Ahh, didn't see this. 😆 
```suggestion
def _test_unicode_category(s: str, categories: Tuple[str, ...]) -> bool:
```

Reply:
Or

```suggestion
def _test_unicode_category(s: str, categories: Sequence[str]) -> bool:
```